modules/gdscript/gdscript_vm.cpp

```diff
@@ -3459,6 +3460,22 @@ Variant GDScriptFunction::call(GDScriptInstance *p_instance, const Variant **p_a
 			}
 			DISPATCH_OPCODE;
 
+			OPCODE(OPCODE_FOR_SECOND_VARIABLE_DICT_TYPE_TEST) {
+				CHECK_SPACE(3);
+
+				GET_VARIANT_PTR(container, 0);
+				GET_VARIANT_PTR(second_variable, 1);
```

Comment:
`second_variable` is unused. Please remove it here, in the codegen (`write_for_dictionary_type_check()`), and in the disassembler. Note that the instruction size will decrease from 3 to 2.